identity/app/controllers/editprofile/ConsentsJourney.scala

```diff
@@ -36,6 +36,10 @@ trait ConsentsJourney
   def displayConsentsJourney(consentHint: Option[String] = None): Action[AnyContent] =
     displayConsentJourneyForm(ConsentJourneyPageDefault, consentHint)
 
+  /** GET /complete-consents */
+  def displayConsentComplete: Action[AnyContent] =
+    displayConsentComplete(ConsentJourneyPageDefault, None)
```

Comment:
What happens if this fails to fetch newsletter subs?

Reply:
This can be currently awkwardly handled as follows: https://github.com/guardian/frontend/blob/010f96ff857600d3b3232109d91a8b4170fa9f4d/identity/app/views/consentJourney.scala.html#L267

Reply:
It will still render the email lists but clicking them wont have any effect

Reply:
@mario-galic @jfsoul maybe I will use the above `if(emailPrefsForm.hasGlobalErrors) { ` and not load the accordian of newsletters if this happens?

Reply:
Updated